docker-compose.yml

```diff
@@ -0,0 +1,108 @@
+version: '3.4'
```

Comment:
These two different files for dev and prod would be hard to keep in sync (eg. this version differs). 

@jacquesbh mentioned this on Slack on Friday:
> yes. And it's just a step away to create a `docker-compose.traefik.yml` and explain that `docker-compose -f docker-compose.yml -f docker-compose.traefik.yml` can be use

Can we use an architecture like this for those extra containers (eg. mailhog, nodejs)?

Reply:
It's not just about additional services...